mesheryctl/cmd/start.go

```diff
@@ -43,14 +45,68 @@ var startCmd = &cobra.Command{
 
 		fmt.Println("Starting Meshery...")
 		start := exec.Command("docker-compose", "-f", dockerComposeFile, "up", "-d")
+		time.Sleep(2 * time.Second)
```

Comment:
This can be done in 2 parts:
- checking if the container is running. For this, I would strongly recommend using the docker Go client and NOT use `docker ps` or something along those lines.
- check if the port 9081 is open and is accepting connections by using the default http client to access `http://localhost:9081`